test/functional/test-gesture-recognizers.js

```diff
@@ -888,7 +888,7 @@ describe('PinchRecognizer', () => {
       {clientX: 120, clientY: 130}]});
     expect(res).to.equal(true);
 
-    gesturesMock.expects('signalEmit_').once();
+    gesturesMock.expects('signalEmit_').twice();
```

Comment:
Should add an `expect` to make sure that it used the correct touch (i.e. it used touches[0] and not the newly added touch).